library/Fission/User/Mutate.hs

```diff
@@ -22,13 +26,28 @@ import qualified Fission.User.Table as Table
 
 -- | Create a new, timestamped entry
 create :: MonadRIO    cfg m
+       => MonadSelda      m
+       => HasLogFunc cfg
+       => SecretDigest
+       -> m (ID User)
+create sekret = do
```

Comment:
💅 we should probably find a new name for `secret` during conflicts. perhaps be more verbose with something like "userSecret" or "secretKey"

Reply:
That's actually residual from Brooke's code. I think "userSecret" is more clear as well. Curious to hear her thoughts

Reply:
Yeah, sure, let's do it 👍 In this case it absolutely makes sense. For a lot of functions, verbose names harm reuse (which is something that you get a LOT more of in Haskell), but this function is pretty concrete ✅

Reply:
👩‍🏫

> "userSecret" or "secretKey"

So, again, I fully agree that `userSecret` makes sense here. Zero argument.

We have more information than just the name: `secret' :: SecretDigest`. Not for this case, because digests make a lot of sense in many places, but if we want to be super clear, it is idiomatic to wrap in a `newtype` wrapper and have `s :: SecretUserKey` or similar.